trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/results/ReqCheck.java

```diff
@@ -182,6 +202,9 @@ public boolean equals(final Object obj) {
 		if (!Arrays.equals(mPeaNames, other.mPeaNames)) {
 			return false;
 		}
+		if(!mMessage.equalsIgnoreCase(other.getMessage())) {

```

Comment:
This violates hashCode contract: things that are equal might have the same hash code. Either also ignore cases during hashcode computation, or compare case sensitive here.